pkg/platform/api/headchef/headchef.go

```diff
@@ -187,11 +187,11 @@ func (r *Client) reqBuildSync(buildReq *headchef_models.V1BuildRequest) (BuildSt
 		payloadType := *created.Payload.Type
 
 		switch payloadType {
-		case headchef_models.BuildStatusResponseTypeBuildCompleted:
+		case headchef_models.V1BuildStatusResponseTypeBuildCompleted:
 			return Completed, created.Payload, nil
-		case headchef_models.BuildStatusResponseTypeBuildFailed:
-			return Failed, created.Payload, locale.WrapError(ErrBuildFailedResp, "headchef_build_failure", "Build Failed: {{.V0}}", created.Payload.Message)
```

Comment:
Note: I removed the `ErrBuildFailedResp`, because the information is already returned via the `Failed` flag.  In the `FetchBuildResult` function, we still want to process the returned information, and it didn't feel right to just remove the error there.